vue-frontend/src/components/ApplyForCareer.vue

```diff
@@ -605,17 +605,19 @@ export default {
 }
 
 .form-control.custom {
-  border: none;
-  border-radius: 0;
-  border-bottom: 1px solid #e2e2e2;
+  border: 1px solid #e2e2e2;
+  border-radius: 10px;
+  box-sizing: border-box;
   box-shadow: none;
   color: #3d3d3d;
   font-size: 1.125rem;
-  padding-left: 0;
+  margin-top: 15px;
+  padding: 10px 16px !important;
```

Comment:
why do we need to add important?

Reply:
Because take a default form-control padding css